src/main/java/dev/jpcode/kits/KitsCommandRegistry.java

```diff
@@ -201,6 +202,27 @@ public static void register(
             ).build()
         );
 
+        kitNode.addChild(literal("commands")
+            .requires(Permissions.require("kits.manage", 4))
+            .then(argument("kit_name", StringArgumentType.word())
+                .suggests(KitsMod::suggestionProvider)
+                .then(literal("list")
+                    .executes(KitCommandsManagerCommand::listCommandsForKit)
+                )
+                .then(literal("add")
+                    .then(argument("command", StringArgumentType.greedyString())
+                        .executes(KitCommandsManagerCommand::addCommandToKit)
+                    )
+                )
+                .then(literal("remove")
+                    .then(argument("command", StringArgumentType.greedyString())
```

Comment:
Can you add this todo? Want to do this eventually, but doesn't have to happen here/now.

```suggestion
                    // TODO: suggestions based on existing commands in kit
                    .then(argument("command", StringArgumentType.greedyString())
```

Reply:
actually, I can just make a github issue, derp: https://github.com/John-Paul-R/kits/issues/18